test/functional/legacy/011_autocommands_spec.lua

```diff
@@ -141,6 +141,7 @@ describe('file reading, writing and bufnew and filter autocommands', function()
   end)
 
   it('FilterReadPre, FilterReadPost', function()
+    if helpers.pending_win32(pending) then return end
```

Comment:
Is there a way to not use sed for this? I can work around `shellescape` but I don't use `sed`.